src/style.css

```diff
@@ -1,5 +1,113 @@
+/* stylelint-disable declaration-block-semicolon-newline-after */
+/* stylelint-disable no-eol-whitespace */
```

Comment:
- [ ] Linters help us write neat and easy-to-read code. Let's maintain that. Kindly remove the linters disabling here and resolve the resulting issues.